vsphere/data_source_vsphere_vmfs_disks.go

```diff
@@ -0,0 +1,95 @@
+package vsphere
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"sort"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/vim25/mo"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+func dataSourceVSphereVmfsDisks() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceVSphereVmfsDisksRead,
+
+		Schema: map[string]*schema.Schema{
+			"host": &schema.Schema{
+				Type:        schema.TypeString,
+				Description: "The name of the host to put this virtual switch on. This is ignored if connecting directly to ESXi, but required if not.",
```

Comment:
is this description right? The datasource name suggest it's about disks, not switches